src/ErrorsNotification.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  error: string,
+  onCloseError: () => void,
+};
+
+export const ErrorsNotification: React.FC<Props> = ({
+  error,
+  onCloseError,
+}) => {
+  const showError = true;
```

Comment:
What's the purpose of this variable? It's constantly true and you don't change it